guides/reference/windows.md

```diff
@@ -1 +1,83 @@
-<!-- Port over info from existing site -->
\ No newline at end of file
+Windows Vista and newer windows versions are fully supported.
+
+To get started ensure the following dependencies are installed:
+
+* Node.js - [https://nodejs.org/en/](https://nodejs.org/en/)
+* Git - [https://git-scm.com/](https://git-scm.com/)
+* Phantom.js - [http://phantomjs.org/](http://phantomjs.org/)
```

Comment:
Is PhantomJS still used on Windows?

https://github.com/emberjs/ember.js/commit/0a5388b28ba81139fec663c792dd062451c08776

Reply:
I don't think so.